### PR TITLE
Add dynamic width modifier for handling Vault setup screen size

### PR DESCRIPTION
**Pull request by aminsato**

### CODE REVIEW

File: app/src/main/java/com/vultisig/wallet/ui/screens/keygen/Setup.kt [hunk 1]
```diff
@@ -104,7 +106,7 @@ internal fun Setup(
                 painter = painterResource(id = state.tabs[state.tabIndex].drawableResId),
                 contentDescription = "devices",
                 contentScale = ContentScale.FillWidth,
-                modifier = Modifier.weight(1f)
+                modifier = Modifier.weight(1f).width(state.tabs[state.tabIndex].imageWidth)
```
[the remark this answers is not in the document]
Comment (aminsato) [reply app/src/main/java/com/vultisig/wallet/ui/screens/keygen/Setup.kt]: done

**Comment by aminsato** [comment]
The reason I resized devices_2_2.webp is that it's malformed on smaller devices.

File: app/src/main/java/com/vultisig/wallet/ui/screens/keygen/Setup.kt [hunk 2]
```diff
@@ -103,8 +105,8 @@ internal fun Setup(
             Image(
                 painter = painterResource(id = state.tabs[state.tabIndex].drawableResId),
                 contentDescription = "devices",
-                contentScale = ContentScale.FillWidth,
-                modifier = Modifier.weight(1f)
+                contentScale = ContentScale.Crop,
```
[the remark this answers is not in the document]
Comment (aminsato) [reply app/src/main/java/com/vultisig/wallet/ui/screens/keygen/Setup.kt]: Is it okay with bigger devices?
ContentScale.Fit
### ScreenShot
Screenshot_20240606_121222

varsus 

ContentScale.Crop
### ScreenShot
Screenshot_20240606_121430

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**